text/0000-rust-governance.md

```diff
@@ -0,0 +1,728 @@
+author and the broader community to dig out problems, alternatives, and improved
+design, always working to reach broader consensus.
+
+At some point, the RFC comments will reach a kind of "steady state", where no
+new tradeoffs are being discovered, and either objections have been addressed,
+or it's clear that the design has fundamental downsides that need to be weighed.
+
+At that point, the shepherd will announce that the RFC is in a "final comment
+period" (which lasts for one week). This is a kind of "last call" for strong
+objections to the RFC. **The announcement of the final comment period for an RFC
+should be very visible**; it should be included in the subteam's periodic
+communications.
+
+> Note that the final comment period is in part intended to help keep RFCs
+> moving. Historically, RFCs sometimes stall out at a point where discussion has
+> died down but a decision isn't needed urgently. In this proposed model, the
+> RFC author could ask the shepherd to move to the final comment period (and
+> hence toward a decision).
+
+After the final comment period, the subteam can make a decision on the RFC. The
+role of the subteam at that point is *not* to reveal any new technical issues or
+arguments; if these come up during discussion, they should be added as comments
+to the RFC, and it should undergo another final comment period.
+
+Instead, the subteam decision is based on **weighing the already-revealed
+tradeoffs against the project's priorities and values** (which the core team is
+responsible for setting, globally). In the end, these decisions are about how to
+weight tradeoffs. The decision should be communicated in these terms, pointing
+out the tradeoffs that were raised and explaining how they were weighted, and
+**never introducing new arguments**.
+
+## Keeping things lightweight
+
+In addition to the "final comment period" proposed above, this RFC proposes some
+further adjustments to the RFC process to keep it lightweight.
+
+A key observation is that, thanks to the stability system and nightly/stable
+distinction, **it's easy to experiment with features without commitment**.
+
+### Clarifying what needs an RFC
+
+Over time, we've been drifting toward requiring an RFC for essentially any
+user-facing change, which sometimes means that very minor changes get stuck
+awaiting an RFC decision. While subteams + final comment period should help keep
+the pipeline flowing a bit better, it would also be good to allow "minor"
+changes to go through without an RFC, provided there is sufficient review in
+some other way. (And in the end, the core team ungates features, which ensures
+at least a final review.)
+
+This RFC does not attempt to answer the question "What needs an RFC", because
+that question will vary for each subteam. However, this RFC stipulates that each
+subteam should set an explicit policy about:
+
+1. What requires an RFC for the subteam's area, and
+2. What the non-RFC review process is.
```

Comment:
At the least it seems that each subteam should have a way to broadcast changes which are major enough to make them nervous, but not major enough to need an RFC - an 'intent to implement' email or something.